# packaging of opencv keypoint object https://docs.opencv.org/3.4/d2/d29/classcv_1_1KeyPoint.html
# this is meant to fullfill the constructor to make a keypoint
# this message is only defined to define other messages, not to be used directly

float32  X
float32  Y
float32  Size
float32  Angle
float32  Response
int32    Octave 
int32    ClassID

